main.js

```diff
@@ -0,0 +1,18 @@
+const navbarHamburger = document.querySelector('#navbar-hamburger');
+const mobileMenu = document.querySelector('#mobile-menu');
+const cross = document.querySelector('#cross');
+navbarHamburger.addEventListener('click', () => {
+  mobileMenu.style.display = 'block';
+});
+
+cross.addEventListener('click', () => {
+  mobileMenu.style.display = 'none';
+});
+
+document.querySelectorAll('.nav-link').forEach(
+  (link) => {
+    link.addEventListener('click', () => {
+      mobileMenu.style.display = 'none';
+    });
```

Comment:
- [ ] Good job so far. However, the mobile menu background should have the same colour as the Figma design. You can use backdrop to have the background blur effect.
_The left one is the figma design and right yours_

<img width="891" alt="Screenshot 2022-10-10 at 4 47 32 PM" src="https://user-images.githubusercontent.com/70269507/194884626-c6a32d56-cf84-4c05-a353-b8165fa61d57.png">

Reply:
@sja-thedude 
I have done that in my CSS. And my background color corresponds to the Figma design if you change the opacity to 1 instead of 0.8
https://github.com/wassimchakib/portfolio-project/blob/3b8e39b4ad9d3709a15f44543464fb670b2350b7/style.css#L116-L117